server/lib/modules/ose-installer/launch.rb

```diff
@@ -55,7 +55,7 @@ def parse
       end
 
       def write_inventory(opts, inventory_file_name = "#{OUTPUT_DIR}/ansible.hosts", template_file_name = "templates/ansible.hosts.template")
-        template = File.read(template_file_name)
+        template = File.read("#{File.dirname(__FILE__)}/#{template_file_name}")
```

Comment:
This is weird why is it a string even though it was already a string. What caused you to use this syntax now?